src/components/ZclEndpointCard.vue

```diff
@@ -21,111 +21,85 @@ limitations under the License.
       @click="setSelectedEndpointType(endpointReference)"
       flat
     >
-      <div
-        class="q-mx-sm"
-        style="display: flex; justify-content: space-between"
-      >
-        <div class="flex q-pa-sm col-4 q-gutter-sm">
-          <img
-            v-if="$store.state.zap.isMultiConfig"
-            :src="
-              createLogoSrc(
-                true,
-                getDeviceCategory(deviceType[0]?.packageRef),
-                isSelectedEndpoint
-              )
-            "
-            alt=""
-          />
-          <strong> #{{ getFormattedEndpointId(endpointReference) }}</strong>
-        </div>
-        <div class="q-gutter-sm" style="display: flex; align-items: center">
-          <q-btn
-            flat
-            dense
-            icon="o_edit"
-            size="sm"
-            v-close-popup
-            @click="modifyEndpointDialog = !modifyEndpointDialog"
-            data-test="edit-endpoint"
-          >
-            <q-tooltip> Edit </q-tooltip>
-          </q-btn>
-          <q-btn
-            flat
-            dense
-            v-close-popup
-            size="sm"
-            icon="o_content_copy"
-            @click.stop="duplicateEndpoint()"
-          >
-            <q-tooltip> Copy </q-tooltip>
-          </q-btn>
-          <q-btn
-            flat
-            dense
-            v-close-popup
-            size="sm"
-            icon="o_delete"
-            @click="handleDeletionDialog"
-            data-test="delete-endpoint"
-          >
-            <q-tooltip> Delete </q-tooltip>
-          </q-btn>
+      <router-link :to="'/'" class="no-decoration">
```

Comment:
not use router-link for this. add @click="" handler and a function for this in a script tag.